Source/platform/ctr/messagebox.cpp

```diff
@@ -0,0 +1,50 @@
+#include <SDL.h>
+#include "utils/sdl2_to_1_2_backports.h"
+#include "utils/display.h"
+#include "utils/ui_fwd.h"
+
+void DisplayMessage(Uint32 flags, const char *title, const char *message)
+{
+	int logCategory = SDL_LOG_CATEGORY_APPLICATION;
+	char fmt[] = "MSGBOX: %s\n%s";
+	if (flags & SDL_MESSAGEBOX_ERROR) {
+		SDL_LogError(logCategory, fmt, title, message);
+		return;
+	}
+	if (flags & SDL_MESSAGEBOX_WARNING) {
+		SDL_LogWarn(logCategory, fmt, title, message);
+		return;
+	}
+	SDL_LogInfo(logCategory, fmt, title, message);
+}
+
+int WaitForJoystickButton()
+{
+	SDL_Event event;
+	while (SDL_WaitEvent(&event)) {
+		if (event.type == SDL_JOYBUTTONDOWN)
+			return 0;
+	}
+
+	SDL_SetError("Unable to receive user input");
+	return -1;
+}
+
+int SDL_ShowSimpleMessageBox(Uint32 flags,
+    const char *title,
+    const char *message,
+    SDL_Surface *window)
+{
+	if (SDL_ShowCursor(SDL_DISABLE) <= -1)
+		SDL_Log("%s", SDL_GetError());
+
+	devilution::SetVideoMode(320, 240, 0, SDL_CONSOLETOP | SDL_BOTTOMSCR);
```

Comment:
I'm assuming that `SDL_CONSOLETOP` means to put it on the top screen?

And combined with SDL_BOTTOMSCR it would put it on the bottom screen.

But isn't that what SDL_CONSOLEBOTTOM does (put the console on the bottom screen)?